app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @recipe.name %></h1>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+
+<h2>Ingredients</h2>
+<ul>
+  <% @ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
+  <% end %>
+</ul>
+
+<p>Total Cost: <%= @total_cost %></p>
```

Comment:
Here is where instead of creating
```ruby
@total_cost = @ingredients.sum(:cost)
```
in your controller, you would have a method called `total_cost` in your `recipe` model and then you would just call `@recipe.total_cost` here.